demo/playground/hmr-playground.tsx

```diff
@@ -14,7 +14,7 @@ const specUrl =
 const options: RedocRawOptions = {
   nativeScrollbars: false,
   maxDisplayedEnumValues: 3,
-  codeSamplesLanguages: ['json', 'xml'],
+  codeSamplesLanguages: ['json', 'xml', 'csv'],
```

Comment:
I don't think so because the constant lives in the `src` folder, whereas this is used for demo / playground purposes, so it shouldn't know about the source code.

Reply:
You are right. Thanks!